conf/pf.conf.defaults

```diff
@@ -35,6 +35,14 @@ timezone=America/Montreal
 # meaning a span port is not necessary.  This feature is highly recommended if the internal network is DHCP-based.
 dhcpdetector=enabled
 #
+# network.dhcp_rate_limiting
+#
+# Will rate-limit DHCP packets that contain the same information.
+# For example, a DHCPREQUEST for the same MAC/IP will only be processed once in the timeframe configured below.
+# This is independant of the DHCP server/relay handling the packet and is only based on the IP, MAC Address and DHCP type inside the packet.
+# A value of 0 will disable the rate limitation.
+dhcp_rate_limiting=5s
```

Comment:
The thing is we could miss some renewals and if someone sets the DHCP lease to 15 seconds, it won't adjust this automatically, so for the default value I wanted to keep something that would work for all the cases and wouldn't create a hard to debug issue.